Radio/example.jsx

```diff
@@ -99,6 +105,13 @@ export default {
         value='ipsum'
       />,
 
+      'Partially collapsed & controlled': <Radio
+        options={optionsWithContent}
+        expandLabel='Show me all the options'
+        collapsedOptions={2}
```

Comment:
Sure. I can invert that API. Sounds reasonable, I'm not sure why I did this in the first place.